commands/compose.go

```diff
@@ -0,0 +1,113 @@
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/docker/model-cli/desktop"
+	"github.com/spf13/cobra"
+)
+
+func newComposeCmd() *cobra.Command {
+
+	c := &cobra.Command{
+		Use: "compose EVENT",
+	}
+	c.AddCommand(newUpCommand())
+	c.AddCommand(newDownCommand())
+	c.Hidden = true
+	c.Flags().String("project-name", "", "compose project name") // unused by model
+
+	return c
+}
+
+func newUpCommand() *cobra.Command {
+	var model string
+	c := &cobra.Command{
+		Use: "up",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if model == "" {
+				err := errors.New("options.model is required")
+				sendError(err.Error())
+				return err
+			}
+
+			client, err := desktop.New()
+			if err != nil {
+				return fmt.Errorf("Failed to create Docker client: %v\n", err)
+			}
+			_, err = client.Pull(model, func(s string) {
+				sendInfo(s)
+			})
+			if err != nil {
+				sendErrorf("Failed to pull model", err)
+				return fmt.Errorf("Failed to pull model: %v\n", err)
+			}
+
+			// FIXME get actual URL from Docker Desktop
+			setenv("URL", "http://host.docker.internal:12434/engines/llama.cpp/v1/")
```

Comment:
```suggestion
			setenv("URL", "http://model-runner.docker.internal:80/engines/v1/")
```

@doringeman that is fine for now, right?

Reply:
```suggestion
			setenv("URL", "http://model-runner.docker.internal/engines/v1/")
```
This is the shortest version.

How's this URL going to be used?